.goreleaser.yml

```diff
@@ -0,0 +1,113 @@
+# .goreleaser.yaml
+project_name: cdk-data-availability
+
+release:
+  disable: false
+  draft: true
+  prerelease: auto
+
+builds:
+  - id: linux-amd64
```

Comment:
This is not necessary, it's building 3 times, only the last one is necessary, and the docker template will pick up the corresponding arch binary.